public/scripts/history.js

```diff
@@ -96,4 +96,17 @@ function recordAnswer(questionNo, timeSpentMS) {
     }
     _history[questionNo].push(timeSpentMS);
     return true;
-}
\ No newline at end of file
+}
+
+module.exports = {
+    getHistory: getHistory,
+    getRemoteHistory: getRemoteHistory,
+    saveRemoteHistory: saveRemoteHistory,
+    loadHistory: loadHistory,
+    saveHistory: saveHistory,
+    recordAnswer: recordAnswer,
+    clearHistory: clearHistory
+
+
+
```

Comment:
An ES6 trick if the key and value are the same, is you can just include the value, e.g.

```js
modules.exports = {
  getHistory,
  getRemoteHistory,
}
```